Gemfile

```diff
@@ -91,12 +65,14 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "database_cleaner"
   gem "capybara-select2"
-  gem "show_me_the_cookies"
-  gem "brakeman"
+  gem 'brakeman', require: false
   gem "pry"
   gem "pry-remote"
   gem 'pry-byebug'
+  gem 'ruby-prof'
+end
+
+group :development do
```

Comment:
Then yeah you guys should have a staging environment for these UAT servers. As rails "test" environment is quite different. It's super easy to set up too. Have a staging.rb in your environments directory (should mimic production as much as possible, maybe with different email configuration and what not) and use a RAILS_ENV=staging instead. I will say though, you should be serving the proper 404.html/500.html error pages on staging like you would on production and not using better_errors?